pkg/services/ngalert/state/manager.go

```diff
@@ -377,6 +377,7 @@ func (st *Manager) saveAlertStates(ctx context.Context, logger log.Logger, state
 			logger.Error("Failed to save alert state", "labels", s.Labels.String(), "state", s.State, "error", err)
 		}
 	}
+	logger.Debug("Saving alert states done", "count", len(states))
```

Comment:
I kept the count so it's easier to correlate the start and end logs when this function is called from multiple goroutines.